### ISSUE TITLE
sub-album support yet?

### ISSUE THREAD

**Issue by imthenachoman** [GustavePate/lycheesync#75]
By reading https://github.com/electerious/Lychee/pull/639 I think Lychee supports sub-albums now. Will lycheesync support it too?

**Comment by GustavePate** [comment]
Yes it will. But not yet, who knows maybe I will have a bit of time to work on it ;)
